core/new-gui/package.json

```diff
@@ -44,6 +44,7 @@
     "@types/gapi.auth2": "~0.0.56",
     "@types/lodash-es": "~4.17.4",
     "@types/mapbox-gl": "~2.6.3",
+    "@types/marked": "^4.0.8",
```

Comment:
lets remove `^` so we pin exact the version

Reply:
sounds good, thanks!